assets/js/wp-hours-widget.js

```diff
@@ -0,0 +1,54 @@
+( function ( $ ) {
+
+	var hoursWidget = {
+
+		init: function( e ) {
+
+			var date        = new Date(),
```

Comment:
@EvanHerman Our date object will need to be instantiated using something like `wpcw_hours.gmt_offset` passed from PHP. Since time blocks are presumably set for the site's timezone, we'll want to use the current time from that timezone when checking for open status.